content/en/blog/posts/2021-12-17-build-release-v0.7.0.md

```diff
@@ -0,0 +1,124 @@
+
+## Revamped Operator Experience
+
+We have completely overhauled the operator for those who use [Operator Lifecycle Manager](https://olm.operatorframework.io/) (OLM) to install Shipwright.
+The operator now comes with its own Custom Resource - `ShipwrightBuild` - that allows administrators to control where the Shipwright build controller is installed.
+It also has the ability to automatically install Tekton Pipelines if Tekton's operator is present in one of OLM's catalogs.
+
+## Dependency updates
+
+We updated our projects to be current: we now build with Go 1.17.
+With [Kubernetes 1.19 being out of support](https://endoflife.date/kubernetes), we changed our minimum version to 1.20 and internally build with the libraries of 1.21.
+We now want you to use [Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.27 at a minimum but recommend to use the latest 0.30 version.
+The Shipwright team contributed a [fix](https://github.com/tektoncd/pipeline/pull/4372) that optimizes reconciliations for TaskRuns that are similar to those created by Shipwright BuildRuns.
+
+## Features
+
+So far, everything was mostly internal. No new features for our users at all? Of course not. :-) But smaller this time:
+
+* We lifted the limit to specify image annotations and labels just on the Build. You can now also define them on the BuildRun. Our CLI finally knows them as well. You can specify them both when creating Builds and BuildRuns.
+* We introduced a new [configuration option on the build controller](https://github.com/shipwright-io/build/blob/v0.7.0/docs/configuration.md): `GIT_ENABLE_REWRITE_RULE`. This is `false` by default, but you can set it to `true` in the deployment. It addresses a problem that many of our users faced downstream when using Shipwright in IBM Cloud Code Engine: to access private code repositories, they managed to get their SSH key into the system and referenced it in the Build. But, they failed to specify the SSH URL but instead used the HTTPS one. The magic flag advises our Git step to setup an [insteadOf](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf) configuration so that the HTTPS URL is internally rewritten to the SSH URL automatically when applicable. The same setting also helps if you have submodules configured using HTTPS in your `.gitmodules` file, but need to pull them with authentication.
+* We worked further on our environment variable support: the Buildpacks sample build strategy now performs the necessary logic to propagate them to the Buildpacks build so that you can customize the behavior of your build using one of the many configurations that the Buildpacks support.
+* We also further enhanced the BuildRun status: for your Git source, we now include the branch name in case you have not specified any revision in your Build. That way you have better insights on what was built.
```

Comment:
should we mention the deprecation of implicit emptydir, with the anticipation of ship-22 and build volumes coming in v0.8.0

Reply:
I'm fine either way, but thought it worth asking

Reply:
We should do it here, too. Not everyone reads release notes.